packages/client/src/views/Home/RecordAudit.tsx

```diff
@@ -0,0 +1,608 @@
+          (application.registrationStatus &&
+            application.registrationStatus.toString()) ||
+          '',
+        type: application.event || '',
+        brnDrn:
+          (application.data.registration &&
+            application.data.registration.registrationNumber &&
+            application.data.registration.registrationNumber.toString()) ||
+          '',
+        trackingId:
+          (application.data.registration &&
+            application.data.registration.trackingId &&
+            application.data.registration.trackingId.toString()) ||
+          '',
+        dateOfBirth:
+          (application.data.child &&
+            application.data.child.childBirthDate &&
+            application.data.child.childBirthDate.toString()) ||
+          '',
+        dateOfDeath:
+          (application.data.deathEvent &&
+            application.data.deathEvent.deathDate &&
+            application.data.deathEvent.deathDate.toString()) ||
+          '',
+        placeOfBirth: getLocation(application, props) || '',
+        placeOfDeath: getLocation(application, props) || '',
+        informant:
+          (application.data.registration &&
+            application.data.registration.contactPoint &&
+            (
+              application.data.registration.contactPoint as IFormSectionData
+            ).value.toString()) ||
+          '',
+        informantContact:
+          (application.data.registration &&
+            application.data.registration.contactPoint &&
+            ((application.data.registration.contactPoint as IFormSectionData)
+              .nestedFields as IContactPoint) &&
+            (
+              (application.data.registration.contactPoint as IFormSectionData)
+                .nestedFields as IContactPoint
+            ).registrationPhone.toString()) ||
+          ''
+      }
+    })
+  return applications[0]
+}
+
+const getWQApplication = (props: IFullProps): IApplicationData | null => {
+  const applicationId = props.match.params.applicationId
+
+  const workqueue = props.workqueue.data
+
+  let applications: Array<
+    GQLBirthEventSearchSet | GQLDeathEventSearchSet | null
+  > = []
+
+  if (workqueue.approvalTab.results)
+    applications = applications.concat(workqueue.approvalTab.results)
+  if (workqueue.printTab.results)
+    applications = applications.concat(workqueue.printTab.results)
+  if (workqueue.inProgressTab.results)
+    applications = applications.concat(workqueue.inProgressTab.results)
+  if (workqueue.externalValidationTab.results)
+    applications = applications.concat(workqueue.externalValidationTab.results)
+  if (workqueue.rejectTab.results)
+    applications = applications.concat(workqueue.rejectTab.results)
+  if (workqueue.reviewTab.results)
+    applications = applications.concat(workqueue.reviewTab.results)
+  if (workqueue.notificationTab.results)
+    applications = applications.concat(workqueue.notificationTab.results)
```

Comment:
Just to be inline with the rest of the codebase

```suggestion
  if (workqueue.approvalTab.results) {
    applications = applications.concat(workqueue.approvalTab.results)
  }
  if (workqueue.printTab.results) {
    applications = applications.concat(workqueue.printTab.results)
  }
  if (workqueue.inProgressTab.results) {
    applications = applications.concat(workqueue.inProgressTab.results)
  }
  if (workqueue.externalValidationTab.results) {
    applications = applications.concat(workqueue.externalValidationTab.results)
  }
  if (workqueue.rejectTab.results) {
    applications = applications.concat(workqueue.rejectTab.results)
  }
  if (workqueue.reviewTab.results) {
    applications = applications.concat(workqueue.reviewTab.results)
  }
  if (workqueue.notificationTab.results) {
    applications = applications.concat(workqueue.notificationTab.results)
  }
```